### ISSUE TITLE
[new design] design-proposal: subhead on scroll

### ISSUE THREAD

**Issue by svom** [joomla/40-backend-template#385]
I would recommend to adjust the styling of the subhead so that it fits better to the header. 
Changes made:
Set the background-color to white, took the box shadow of the header, adust width to calc(100+30px), margin-left: -15px; increased horizontal-padding.

Current state:
![image](https://user-images.githubusercontent.com/14434412/38813431-2ea1c528-418f-11e8-9963-c40a4aef3b95.png)

Proposal:
![image](https://user-images.githubusercontent.com/14434412/38813447-3a6d7460-418f-11e8-8f6e-1a11c80913b1.png)

**Comment by dgrammatiko** [comment]
It requires intersection observer which is not widely supported: https://caniuse.com/#feat=intersectionobserver

But it's polyfillable: https://github.com/w3c/IntersectionObserver/tree/master/polyfill

There is one thing that is lose ATM: we don't have a definite way of dealing with polyfills in J4. That's something that was discussed few times in the JS group but we still don't have a resolution or any code...

**Comment by dgrammatiko** [comment]
Just to add that next version of Safari will also support it:
https://webkit.org/status/#?search=intersection

**Comment by brianteeman** [comment]
iirc the status "in development" means that it is for the version after the next as it moves from "supported in preview" to "release"

anyway it doesnt matter as we would require a polyfill anyway for ie11

**Comment by dgrammatiko** [comment]
You're right, todays release of TP doesn't support it, so probably it will take some more months...

**Comment by brianteeman** [comment]
I am always right it just takes some of you a while to realise ;)
